src/packages/admin-ui-components/src/popover/component.tsx

```diff
@@ -0,0 +1,105 @@
+import { ReactNode, useState, useMemo, useCallback } from 'react';
+
+import { Button } from '../button';
+import type { ButtonProps } from '../button';
+import { ReactComponent as DownChevronIcon } from '../assets/16-chevron-down.svg';
+
+import styles from './styles.module.css';
+import classNames from 'classnames';
+
+export interface PopoverItem {
+	id: string /** Unique ID for this item */;
+	href?: string /** href to link to */;
+	name: string /** Label to appear for the Popover item */;
+	onClick?():
+		| void
+		| boolean /** Event emitted when the Popover item is clicked- return false to prevent closing the Popover  */;
+	renderAfter?(): ReactNode /** Render function after the item */;
+	className?: string;
+}
+
+export interface PopoverProps extends Partial<ButtonProps> {
+	items: Array<PopoverItem> /** List of items in the Popover */;
+	className?: string /** Make button look like textfield not button */;
+	defaultValue?: PopoverItem;
+	position?: 'top' | 'bottom';
+}
+
+export const Popover = ({
+	items,
+	children,
+	defaultValue,
+	position = 'bottom',
+	...props
+}: PopoverProps): JSX.Element => {
+	const [isOpen, setIsOpen] = useState(false);
+
+	function handleOnClickItem(item: PopoverItem) {
```

Comment:
Any reason for not using arrow function here? and you could also collapse the return?

Reply:
Yea this is just how it was before. Will convert.